_posts/blog/2016-05-03-droidcon_zagreb.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Droidcon Zagreb 2016"
+modified:
+categories: blog
+author: florina_muntenescu
+excerpt: An overview of the things learned and best talks at Droidcon Zagreb 2016
+tags: [Conference, RxJava, MVVM]
+image:
+date: 2016-05-2T15:39:55-04:00
+---
+
+Going to conferences is great - you get to learn new things and meet new people. But going as a speaker is even better! It feels like you’re doing your duty as a good developer by also giving something back to the community. This is something that we want to do more at upday.
+Here’s an overview of some of the most interesting things I’ve learned at Droidcon Zagreb. Make sure you checkout the linked presentations.
+
+## Latest from Google
+
+The first day’s keynote from Wojtek Kaliciński brought a lot of insights into what’s new in Android Studio, in Google Play and about what we can expect from Android N.
+
+### Google Play
+Google Play has a lot of features that most of the developers probably don’t know about. One example would be <a href="https://developers.google.com/analytics/devguides/collection/android/v4/">Google Analytics</a>, that has a large range of tracking related functionalities, that are being constantly improved. Google offers a wide range of <a href="https://developers.google.com/products/">services for developers</a>. Make sure you check those out before you decide on any other 3rd party tools.
+
+### Architecture Samples
+For anyone that wonders how they should architect their projects, Google has <a href="https://github.com/googlesamples/android-architecture">Android architecture samples</a> in their GitHub repo. Examples using popular libraries like Dagger 2 and RxJava are also included.
+
+### Android N
+Although the preview of Android N was already released and some of its features already made public in the <a href="http://developer.android.com/preview/behavior-changes.html">behavior changes page</a>, it was great hearing more about them. The system activity restrictions brought by **Doze**, now have two stages and are for sure something to keep in mind when developing applications that need to run background tasks and handle Google Cloud Messages.
```

Comment:
Doze and multi window support are bold since these are the main topics to keep in mind from Android N.